pipelines/simple-notebook-pipeline/Simple Notebook Pipeline.ipynb

```diff
@@ -213,7 +170,7 @@
    "source": [
     "#### Create an Experiment in the Pipeline System\n",
     "\n",
-    "Pipeline system requires an \"Experiment\" to group pipeline runs. You can create a new experiment, or call client.list_experiments() to get existing ones."
+    "Kubeflow Pipelines lets you group pipeline runs by *Experiments*. You can create a new experiment, or call `client.list_experiments()` to see existing ones."
```

Comment:
Let's replace create_experiment+compile+run_pipeline with `client.create_run_from_pipeline_func`, like in the `kubeflow/pipelines`' samples.